dutch/bos/donker-bos.md

```diff
@@ -4,3 +4,5 @@ Je verstaat geen woord ervan, maar de stemmen komen wel steeds sneller naar je t
 Je verstopt je in de struiken, en ziet twee mannen met een lijkzak sleuren.
 
 Je gaat snel [terug naar de hei](../welkom.md)
+
+Je gaat [de confrontatie](../confrontatie/confrontatie.md) aan, ookal heb je enkel een slappe zwemnoodle om je te verdedigen. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.